src/main/java/Technique.java

```diff
@@ -0,0 +1,155 @@
+            ": Si une île a un chiffre impair > 1 elle à au minimum n%2+1 voisins. Si elle a " +
+            "exactement ce nombre de voisins alors elle est obligatoirement relié au moins une fois à " +
+            "chacun de ses voisins. Si parmi ses voisins il y a une île avec le chiffre 1, alors l'île sera relié 2 fois à " +
+            "tous ses autres voisins"
+            , o -> {
+        int n = o.getN();
+        int nbV = o.nbVoisin();
+        int nbV1 = o.nbVoisinAvec1();
+        if(n == 7 && nbV1 == 1) return true;
+        if(n == 5 && nbV == 3 && nbV1 == 1) return true;
+        if(n == 3 && nbV == 2 && nbV1 == 1) return true;
+        return false;
+    }),
+    TECH_BAS_3("Technique 4",
+            "Si une île a le chiffre 1 ou 2 et qu’elle possède seulement 1 voisin alors elle sera " +
+            "forcément reliée à ce voisin, cela fonctionne seulement le chiffre 1 et 2 puisque dans le " +
+            "hashi il peut seulement exister maximum 2 ponts entre 2 îles."
+            , o -> {
+        int n = o.getN();
+        int nbV = o.nbVoisin();
+        if(n == 1 || n == 2){
+            return nbV == 1;
+        }
+        return false;
+    }),
+    TECH_BAS_4("Technique 5",
+            "Si une île de valeur 4 possède trois voisins, " +
+            "dont deux d'entre eux sont de valeurs 1, alors on peut la compléter. " +
+                    "Même chose si une île est de valeur 5, avec trois voisins de valeur 1." ,
+            o -> {
+        return ((o.getN() == 4 && o.nbVoisin() == 3 && o.nbVoisinAvec1() == 2) || (o.getN() == 5 && o.nbVoisin() == 4 && o.nbVoisinAvec1() == 3));
+    }),
+    TECH_BAS_5("Technique 6",
+            "Si une île de valeur 6 possède un voisin de valeur 1, alors il possède au moins " +
+                    "un pont avec chacun de ses autres voisins."
+            , o -> {
+        boolean verif = false;
+        if (o.getN() == 6 && o.nbVoisinAvec1() == 1){
+            Direction[] lD = Direction.values();
+            for(int i=0; i<4; i++){
+                verif = verif || (o.valeurIleDirection(lD[i]) != 1 && o.nbPontsDirection(lD[i]) == 0);
+            }
+        }
+        return verif;
+    }),
+    TECH_ISO_1("Technique 7",
+            "Si une île de valeur 1 est voisin avec un 1, il ne peut pas y avoir de pont " +
+                    "entre eux, car cela brise la règle d'isolation."
+            , o -> {
+        return(o.getN() == 1 && (o.nbVoisin()-o.nbVoisinAvec1() == 1));
+    });
+
+    private final String nom;
+    private final String description;
+    private final Function<Ile, Boolean> fonction;
+
+    Technique(String nom, String description, Function<Ile, Boolean> fonction) {
+        this.nom = nom;
+        this.description = description;
+        this.fonction = fonction;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public Function<Ile, Boolean> getFonction() {
+        return fonction;
+    }
```

Comment:
Manque de la documentation ici,

De plus cette méthode peut devenir quelque chose de ce genre:

```suggestion
    public boolean execute(Ile ile) {
        return function.apply(ile);
    }
```